examples/AccountManagement/LinkManagerToClient.php

```diff
@@ -0,0 +1,259 @@
+     * but that may not be the case. If you need to use different credentials
+     * for each customer, then you may either update the client configuration or
+     * instantiate the clients accordingly, one for each set of credentials. Always make
+     * sure to update the configuration before fetching any services you need to use.
+     *
+     * @param int $managerCustomerId the manager customer ID
+     * @param int $clientCustomerId the client customer ID
+     */
+    public static function runExample(int $managerCustomerId, int $clientCustomerId)
+    {
+        // Extends an invitation to the client while authenticating as the manager.
+        $managerLinkResourceName = self::createInvitation($managerCustomerId, $clientCustomerId);
+
+        // Accepts the manager's invitation while authenticating as the client.
+        self::acceptInvitation($clientCustomerId, $managerLinkResourceName);
+    }
+
+    /**
+     * Extends an invitation from a manager customer to a client customer.
+     *
+     * @param int $managerCustomerId the manager customer ID
+     * @param int $clientCustomerId the client customer ID
+     * @return string the resource name of the manager link created for the invitation
+     */
+    private static function createInvitation(
+        int $managerCustomerId,
+        int $clientCustomerId
+    ) {
+        // Creates a client with the manager customer ID as login customer ID.
+        $googleAdsClient = self::createGoogleClient($managerCustomerId);
+
+        // Creates a customer client link.
+        $customerClientLink = new CustomerClientLink([
+            // Sets the client customer to invite.
+            'client_customer' =>
+                new StringValue(['value' => ResourceNames::forCustomer($clientCustomerId)]),
+            'status' => ManagerLinkStatus::PENDING
+        ]);
+
+        // Creates a customer client link operation for creating the one above.
+        $customerClientLinkOperation = new CustomerClientLinkOperation();
+        $customerClientLinkOperation->setCreate($customerClientLink);
+
+        // Issues a mutate request to create the customer client link.
+        $customerClientLinkServiceClient = $googleAdsClient->getCustomerClientLinkServiceClient();
+        $response = $customerClientLinkServiceClient->mutateCustomerClientLink(
+            $managerCustomerId,
+            $customerClientLinkOperation
+        );
+
+        // Prints the result.
+        $customerClientLinkResourceName = $response->getResult()->getResourceName();
+        printf(
+            "An invitation has been extended from the manager customer %d" .
+            " to the client customer %d with the customer client link resource name '%s'.%s",
+            $managerCustomerId,
+            $clientCustomerId,
+            $customerClientLinkResourceName,
+            PHP_EOL
+        );
+
+        // Creates a query that retrieves the manager link ID of the customer client link
```

Comment:
In all examples, when we do a mutate operation and then a query to retrieve the result, the query is in a separate function. Can you please extract this to a separate function as well? Not just for coherence with the other examples, but also because you now have a 68-line function, which looks pretty long to me.

Reply:
+1